utils/vision_utils.py

```diff
@@ -0,0 +1,95 @@
+import time
+import logging
+from datetime import datetime
+import os
+from typing import List
+
+logger = logging.getLogger(__name__)
+
+
+def validate(val_loader, model, criterion, args):
+    from trailmet.utils import accuracy, AverageMeter
+    import torch
+    logger = logging.getLogger(__name__)
```

Comment:
we need not create another logger instance. we already have one instance at line 7. one of these can be removed.